CONTRIBUTING.md

```diff
@@ -53,11 +54,8 @@ There are some external applications loading resources from http://sourced.tech/
 - `https://sourced.tech/img/logos/favicon.png`
 
 #### Linked pages
-- `https://sourced.tech/engine`
-- `https://sourced.tech/lookout`
-- `https://sourced.tech/open-source`
-- `https://sourced.tech/community`
-- `https://sourced.tech/company`
+
+- Current Header and Footer Navigation
```

Comment:
The goal of this section is to know which links to our Landing appear in Pardot pages (in the footer, the header, but and also in their main content).
That's necessary to have an inventory of the links that should be kept in Landing to avoid having broken links in Pardot.